example/counterBasedRng/CMakeLists.txt

```
@@ -33,6 +33,11 @@ if(NOT TARGET alpaka::alpaka)
     endif()
 endif()
 
+if (alpaka_USE_MDSPAN STREQUAL "OFF")
```

Just a hint: You could also do this:

```suggestion
if (NOT alpaka_USE_MDSPAN)
```

---

Are you sure? `alpaka_USE_MDSPAN` can have the values `OFF`, `SYSTEM` or `FETCH`.